shared/src/libs/helpers/replace-template-with-value/replace-template-with-value.helper.ts

```diff
@@ -0,0 +1,13 @@
+const replaceTemplateWithValue = (
+  template: string,
+  replacements: Record<string, string | number>,
+): string => {
+  let result = template;
+  for (const [placeholder, value] of Object.entries(replacements)) {
+    result = result.replaceAll(`{${placeholder}}`, String(value));
+  }
```

Comment:
Let's use .reduce here

Reply:
@v0ldemar01 - we have a rule against using it `Array#reduce()` is not allowed eslint[unicorn/no-array-reduce]
<img width="1096" alt="Screenshot 2023-09-01 at 10 16 52" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/90211108/d4ff92d6-324c-45d9-8b8f-03d2bd5fcf42">

Reply:
Got it, sorry